polygon.go

```diff
@@ -39,6 +39,11 @@ func (p *Polygon) IsClosed() bool {
 		return false
 	}
 
+	// !Currently the following section would break to much older code as it is more strict on the definition of a closed polygon.
+	// if *p.points[0] != *p.points[len(p.points)-1] {
+	// 	return false
+	// }
+
```

Comment:
Interesting insight here. 

If it's not being used, would you mind removing the commented out code?  While it's good to know the implementation, it might confuse future maintainers if the implementation was intended to be in the code path in the first place.

Reply:
Done!

Reply:
```suggestion

```